eng/common/scripts/check-spelling-in-changed-files.ps1

```diff
@@ -79,17 +86,28 @@ Set-Content `
     -Path $cspellConfigTemporaryPath `
     -Value (ConvertTo-Json $cspellConfig -Depth 100)
 
+# cspell will merge configurations from files in well-known locations. In this
+# case the `.vscode/cspell.json` file is a well-known location. To force cspell
+# to use ONLY the temporary config file we rename the config file in the
+# well-known location to a temporary name. (`.vscode/cspell.json.ignore`)
+Write-Host "Renaming $CspellConfigPath"
```

Comment:
Given you are already touching this file should we instead just overwrite it with our update and then undo that overwrite (perhaps using git to undo) when we are done?

Reply:
Given that we're already using git... this would be a safe assumption to make.

Reply:
As we chatted lets not use git but we can keep the content we read first and then overwrite the config file once we are done running cspell.